doc/contributing.rst

```diff
@@ -216,7 +217,8 @@ Install the database
 
 .. code-block:: bash
 
-    elabctl mysql
+    cd ..
```

Comment:
maybe add a comment on top: "go back to where `elabctl` is present". And maybe it is best to use an explicit "cd $dev" that will work everywhere everytime.